test/js/node/stream/node-stream.test.js

```diff
@@ -336,7 +337,7 @@ describe("TTY", () => {
 });
 `;
 
-it("TTY streams", () => {
+it.skipIf(isWindows)("TTY streams", () => {
   mkdirSync(join(tmpdir(), "tty-test"), { recursive: true });
```

Comment:
☝️ TODO: comment to get back to it please (unless it's just not a windows thing)